samples/Calling/src/app/views/CallCompositeContainer.tsx

```diff
@@ -19,7 +19,10 @@ export const CallCompositeContainer = (props: CallCompositeContainerProps): JSX.
   const options: CallCompositeOptions = useMemo(
     () => ({
       onPermissionsTroubleshootingClick,
-      onNetworkingTroubleShootingClick
+      onNetworkingTroubleShootingClick,
+      callControls: {
+        legacyControlBarExperience: false
```

Comment:
Is this the option to get back to the old control bar?

Reply:
Yep, once this is true, the old control bar will be showing up